### PR TITLE
Feature/ppcdev 8176 step by step guide

### PR DESCRIPTION
**Pull request by plks**
step by step guide -> curl + Postman example

### CODE REVIEW

File: platform/authorized_api_guide.md [hunk 1]
```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
   Name must contains at least 3 characters.
 * Copy your newly generated `CLIENT ID` and `CLIENT SECRET` because they **won't be available for you after dismissing this window.**
 
-### Create access token
+This credentials will be valid as long as you revoke them in your profile.
```
Comment (RomanSek) [inline platform/authorized_api_guide.md]: I have problem with understanding this sentence:
* Linguistic: "This credential ..." or "Those credentials ..." are correct, "This credentials ..." is not.
* Logic: Credentials will be valid if I'll revoke them or as long as I wont revoke them?

File: platform/authorized_api_guide.md [hunk 2]
```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
...
+Response example:
 ```
-POST /auth/token
+{"token_type":"Bearer","expires_in":86400,"access_token":"<your_access_token>"}
 ```
 
-Curl example:
+Now, you can use obtained `<your_access_token>` for communication with Piwik PRO API.
+Field `expires_in` stands for time (in seconds) for token expiration (TTL).
+As token is a Bearer type, it must be **included in every API call** within header.
```
Comment (RomanSek) [inline platform/authorized_api_guide.md]: Maybe change it to: "Since token ..."?

File: platform/authorized_api_guide.md [hunk 3]
```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
...
+Whatever API call you choose, first remember that you must generate 
+[API credentials](#generate-api-credentials) for obtaining client id and secret.
+
+### API usage example with curl
+
+For sake of this examples, `https://<domain>` is a URL of your PPMS instance (e.g. `https://example.piwik.pro`) and our goal
+is to perform basic operations on an app. We will:
+* create an app
+* get created app
+* update its some fields
```
Comment (RomanSek) [inline platform/authorized_api_guide.md]: I don't get it. Should this be "update its fields/attributes"?

File: platform/authorized_api_guide.md [hunk 4]
```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
...
+* in the end, we will remove given app
```
Comment (RomanSek) [inline platform/authorized_api_guide.md]: Just "remove the app" should be enough in this point.

File: platform/authorized_api_guide.md [hunk 5]
```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
...
+```
+{
+   "data":{
+      "type":"ppms\/app",
+      "id":"b30e538d-4b05-4a75-ae25-7eb565901f38",
+      "attributes":{
+         "name":"AppName",
+         "addedAt":"2018-09-13T12:16:30+00:00",
+         "urls":[
+            "http:\/\/example.com"
```
Comment (RomanSek) [inline platform/authorized_api_guide.md]: Is this correct? I don't think JSON escapes slashes, but it shouldn't affect build docs. It's just harder to read in the source.

File: platform/authorized_api_guide.md [hunk 6]
```diff
@@ -20,40 +21,317 @@ API credentials which then allows you to request for a token that is used for au
...
+
+Simply click in Postman: `import -> Import From Link`. Then all of your paths are imported!
+You have to override two things:
+* replace your domain in url
+* add token. Click on `Authorization` tab on chosen API call and then use Bearer Token type. 
+  Paste your token and now you can call API using `SEND` button.
+
+## FAQ
+
+Here you can find most common issues during work with API.
```
Comment (RomanSek) [inline platform/authorized_api_guide.md]: Maybe "Here you can find the most common issues encountered during work with the API"?

File: platform/authorized_api/users/index.json [hunk 7]
```diff
@@ -397,6 +417,31 @@
...
+    },
+    "/api/users/v2/me/details": {
+      "get": {
+        "tags": [
+          "Users v2"
+        ],
+        "summary": "Current user details",
+        "operationId": "api_user_details_v2",
+        "produces": [
+          "application/vnd.api+json"
```
[the remark this answers is not in the document]
Comment (plks) [reply platform/authorized_api/users/index.json]: no, sometimes we return `204` without Content Type sometimes we produce it. It cannot be generalized as `consumes` section

File: platform/authorized_api_guide.md (hunk 5 above)
Comment (plks) [reply platform/authorized_api_guide.md]: yes it is semantically correct, but as you pointed it's not needed

**Pull request merged**